File: 1950-sign-of-the-product-of-an-array/1950-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        k=1
        for i in nums:
            if(i==0):
                return 0
            if(i<0):
                k*=-1
            if(i>0):
                k*=1
        if(k<0):
            return -1
        return 1
        

File: 1950-sign-of-the-product-of-an-array/1950-sign-of-the-product-of-an-array.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        long long k=1;
        for (int i=0;i<nums.size();i++){
            if(nums[i]<0){
                k*=-1;
            }
            if(nums[i]>0){
                k*=1;
            }
            if(nums[i]==0){
                return 0;
            }
        }
        if(k<0){
            return -1;
        }
        if(k>0){
            return 1;
        }
        return 0;
        
    }
};